### ISSUE TITLE
Documentation of vg= and metavg=

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#866]
Originally reported of Google Code with ID 810.

```
[ganeti 2.10.2]

The gnt-instance manpage describes the following parameters when adding a disk:

       vg     The LVM volume group.  This works only for LVM and DRBD devices.

       metavg This options specifies a different VG  for  the  metadata  device.
              This works only for DRBD devices

The phrase "specifies a different VG for the metadata device" suggests to me that the metadata device would otherwise get the same VG as the data device. But what actually happens is it gets the default metavg for the cluster (or node group).

Suggested wording:

       vg     The LVM volume group.  This works only for LVM and DRBD devices.

       metavg This option specifies the VG for the metadata device.
              This works only for DRBD devices. If not specified, the node-group or
              cluster-wide "metavg" setting for the drbd disk template is used.

```
Originally added on 2014-04-21 21:16:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-04-22 11:19:17 +0000 UTC.

Added Labels: Type-Documentation
Changed State: Accepted
Added to Milestone: Release2.10

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:

commit 9024059109a035629d6dd5aa29419828f0ed5656
Author: Helga Velroyen <helgav@google.com>
Date:   Tue Apr 22 13:23:50 2014 +0200

    Clarify default setting of 'metavg'
    
    This fixes issue 810, suggesting to clarify where the
    default for 'metavg' comes from.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Hrvoje Ribicic <riba@google.com>

```
Originally added on 2014-04-22 12:02:36 +0000 UTC.

Mentioned Issue with GoogleCode ID 810 has been migrated to GitHub with ID #866.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in 2.10.4.
```
Originally added on 2014-05-15 10:36:57 +0000 UTC.

Changed State: Released
